spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java

```diff
@@ -445,21 +425,11 @@ private void examineCallSite(Location location, ConstantPoolGen cpg, TypeDataflo
             }
         }
          */
-        BitSet nullArgSet = frame.getArgumentSet(invokeInstruction, cpg, new DataflowValueChooser<IsNullValue>() {
-            @Override
-            public boolean choose(IsNullValue value) {
-                // Only choose non-exception values.
-                // Values null on an exception path might be due to
```

Comment:
Sorry, I mean of course full comment, not just the first line. Also the line below :-)

Reply:
Eclipse relaunched... ;)